incsrc/Algebra.cpp

```diff
@@ -53,3 +53,12 @@ Vec& Vec::operator=(Vec& vec)
         data[i] = vec.data[i];
     return *this;
 }
+
+Vec& Vec::operator=(Vec&& vec)
+{
```

Comment:
please comment why you made these changes for future references

Reply:
Will do that